### PR TITLE
upload files

### PR DESCRIPTION
**Pull request by despe-reymondchristian**
Adding my files that is needed to main branch.

### CODE REVIEW

File: despe_reymondchristian/despe_reymond.html [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+				</div>
+				<div class="form-container">
+					<div class="message-form">
+						<form action="">
+							<label for="name">Name:</label>
+							<input type="text" 
+							id="name" 
+							oninput="onTextChange()" 
+							placeholder="Enter your name here..">
+							<br>
```
Comment (Lopez-JhonLay) [inline despe_reymondchristian/despe_reymond.html]: remove the br tag

File: despe_reymondchristian/script.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
+function onTextChange() {
+    let message = document.getElementById("message");
+    let name = document.getElementById("name");
+    let commentButton = document.getElementById("comment_button")
+
+    if (message.value.length > 0 && name.value.length > 0) {
```
Comment (Lopez-JhonLay) [inline despe_reymondchristian/script.js]:
```suggestion
    if (message.value.length && name.value.length) {
```

File: despe_reymondchristian/style.css [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+  width: 150px;
+  margin: 32px 0;
+}
+
+.message-form form {
+  display: flex;
+  flex-direction: column;
+}
+
+#name, #message {
```
Comment (Lopez-JhonLay) [inline despe_reymondchristian/style.css]:
```suggestion
#name,
#message {
```

**Review by Lopez-JhonLay — CHANGES_REQUESTED** [review]
please see suggested changes

File: despe_reymondchristian/script.js [hunk 4]
```diff
@@ -0,0 +1,23 @@
+function onTextChange() {
+    let message = document.getElementById("message");
```
Comment (Intia-JoannaMarie) [inline despe_reymondchristian/script.js:2]:
```suggestion
    let message = document.querySelector("#message");
```

File: despe_reymondchristian/script.js [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+    let name = document.getElementById("name");
```
Comment (Intia-JoannaMarie) [inline despe_reymondchristian/script.js:3]:
```suggestion
    let name = document.querySelector("#name");
```

File: despe_reymondchristian/script.js [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+    let commentButton = document.getElementById("comment_button")
```
Comment (Intia-JoannaMarie) [inline despe_reymondchristian/script.js:4]:
```suggestion
    let commentButton = document.querySelector("#comment_button")
```

**Review by Intia-JoannaMarie — CHANGES_REQUESTED** [review]

**Review by Lopez-JhonLay — COMMENTED** [review]
good job

**Review by Intia-JoannaMarie — COMMENTED** [review]
Nice

**Pull request merged**